npm/webpack-preprocessor/test/unit/index.spec.js

```diff
@@ -151,6 +151,20 @@ describe('webpack preprocessor', function () {
         })
       })
 
+      it('adds output options when user output options are non-default', function () {
```

Comment:
This test doesn't really check that when we have dynamic import somewhere that webpack doesn't crash. It only checks that specific options passed to webpack config. 
What do you think about adding more complex test that shows the desired behavior if @cypress/react tests won't work?